controller/ItemImportSampleDownload.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * DownloadSampleCsv Controller provide actions to download sample CSV
```

Comment:
Could you please use following pattern of copyrigth headers https://github.com/oat-sa/extension-tao-itemqti/pull/1751/files#diff-011a17d61ee057fb83fa3032ad1b8108034ecae3b80a18bb9072c9fa38a82b10R4 everewhere you are about to add new one?